test/e2e/env.rb

```diff
@@ -4,7 +4,7 @@
 # secrets 🤫
 ENV['TESTS_E2E_FIXTURES'] ||= 'this_is_wrong_secret'
 ENV['TESTS_E2E_FIXTURES_FILE'] ||= './fixtures/fixture_wallets.json'
-ENV['TESTS_E2E_BLOCKFROST_FILE'] ||= './fixtures/blockfrost.api.key'
+ENV['BUILDKITE_API_TOKEN'] ||= 'this_is_wrong_secret'
```

Comment:
Is this used at all?

Reply:
can be, if you have no env variable defined.

Reply:
Done.